connection_pool/const.go

```diff
@@ -0,0 +1,47 @@
+package connection_pool
+
+type Mode uint32
+type Role uint32
+type State uint32
+
+/*
+Mode parameter:
```

Comment:
Such lists are parsed a bit ugly by godoc.
![image](https://user-images.githubusercontent.com/20455996/164530664-fddcf010-857a-4186-b1c2-f9e54f8e3312.png)

See https://github.com/tarantool/go-tarantool/pull/151#discussion_r842921642

Reply:
Btw, please do this to all lists in doc comments 
![Uploading image.png…]()

Reply:
Done.